overlord/snapstate/snapmgr.go

```diff
@@ -484,6 +483,18 @@ func (m *SnapManager) ensureRefreshes() error {
 		return err
 	}
 
+	when := timeutil.Next(refreshSchedule, lastRefresh)
+
+	m.nextRefresh = time.AfterFunc(when, func() { m.doAutoRefresh(updated, tasksets) })
+	logger.Debugf("schedule next refresh in %s", when)
```

Comment:
Phrasing is a bit curious. Perhaps `("Next refresh scheduled for %s.", time.Now().Add(delta))`